### PR TITLE
Adding Pratyush

### PR DESCRIPTION
**Pull request by pratyushpal**

### CODE REVIEW

File: src/app/team-page/team-page.component.html [hunk 1]
```diff
@@ -83,6 +83,14 @@
         <p class="title">project developer</p>
       </div>
     </div>
+    <div class="row>"
+    <div class="col-md-3 col-sm-6">
```
Comment (chrisjluc) [inline src/app/team-page/team-page.component.html]: Can you indent line 87 - 93 two spaces to the right?!

Comment (chrisjluc) [inline src/app/team-page/team-page.component.html]: Also add a new line between line 85 and 86

**Comment by hkjayakumar** [comment]
@pratyushpal Please change the base branch to dev!

**Comment by hkjayakumar** [comment]
@pratyushpal can you create a new PR to keep it clean?

**Comment by pratyushpal** [comment]
@hkjayakumar sure

**Pull request closed without merging**